tic-tac-toe-python/GameEngine.py

```diff
@@ -0,0 +1,57 @@
+import re
+from TicTacToe import TicTacToe
+from Player import Player
+
+class GameEngine:
+    def __init__(self):
+        self.game=TicTacToe()
+        self.turn=0
+        self.symbol=['X','O']
+        self.player_x=Player('X')
+        self.player_o=Player('O')
+    
+    def inputValidator(self,inputString):
+        inputRegex = re.compile(r'^\d,\d$')
+        matchedInput=inputRegex.search(inputString)
+        
+        if matchedInput==None:
+            return False
+        return True
+    
+    def inputCoordinates(self):
```

Comment:
function in snake_case

Reply:
All functions and variables should be in snake_case and classes in camelCase right ?